test-kuttl/e2e/restic-with-restoreasof/22-recreate-restore.yaml

```diff
@@ -0,0 +1,7 @@
+# issues a command to recreate the restic snapshot
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - timeout: 2400
```

Comment:
Why would this step take a long time?

Reply:
In the conversation the other day it was mentioned that we needed to bump up the timeouts, at the time I was under the impression that it was the cause of my e2e test failures.